components/dialog/overview.md

```diff
@@ -53,6 +53,12 @@ The Dialog component and its [predefined options]({%slug dialog-predefined%}) ai
 
 Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
 
+## Integration With Other Components
+
+You can use other components inside the `DialogContent`. In this case, the two-way binding for the parameters of these components may not work as expected. The Dialog needs to [`Refresh`](#dialog-reference-and-methods) to reflect UI changes. You can invoke the Dialog's `Refresh` method:
+* When using two-way binding - in the `OnChange` event of the component, used as `DialogContent`.
```

Comment:
Replace the hashes with em dashes in all list items. See the [Punctuation](https://docs.telerik.com/style-guide/punctuation#lists) topic.